src/App.tsx

```diff
@@ -1,156 +1,93 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useMemo, useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Header } from './components/Header';
+import { ErrorMessage } from './components/ErrorMessage';
+import { StatusFilterType } from './types/StatusFilterType';
+import { StatusFilter } from './components/StatusFilter';
 
-const USER_ID = 0;
+const USER_ID = 10897;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<string | null>(null);
+  const [filter, setFilter] = useState(StatusFilterType.ALL);
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then((todosFromServer) => setTodos(todosFromServer))
+      .catch(() => setError('Loading todos failed'));
+  }, []);
+
+  useEffect(() => {
+    let timeout: NodeJS.Timeout;
+
+    if (error) {
+      timeout = setTimeout(() => {
+        setError(null);
+      }, 3000);
+    }
+
+    return () => {
+      clearTimeout(timeout);
+    };
+  }, [error]);
+
   if (!USER_ID) {
     return <UserWarning />;
   }
 
+  const uncompletedTodos = todos.filter(todo => !todo.completed);
+  const completedTodos = todos.filter(todo => todo.completed);
+
+  // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
Why?

Reply:
`// eslint-disable-next-line react-hooks/rules-of-hooks`
had some kind of labels problem, idk any other solution

Reply:
@PANCH1811 that's not about labels problems here

Reply:
understood, fixed ;)